src/components/coming-soon/coming-soon.css

```diff
@@ -0,0 +1,62 @@
+@import "../../css/colors.css";
+
+.coming-soon,
+.coming-soon:before,
+.coming-soon:after {
+    opacity: 1 !important;
+}
+
+.coming-soon {
+    background-color: $data-primary !important;
```

Comment:
I've always wondered if the exclamation point is meant to mean "seriously!", in a code context it kind of looks like "NOT important". 

But actually, what is with all the importants? Could we solve it with some different scoping? Maybe not, just figured I'd ask because this file made my CSS senses ~~tingle~~ light on fire.

Reply:
@paulkaplan I also find the `important`s not ideal. it’s unfortunately the way the third party library is [set up](https://github.com/wwayne/react-tooltip#options), requiring them to override its default styling.